### ISSUE TITLE
[Android] Duration cannot be updated in notification.

### ISSUE THREAD

**Issue by Asoundaccount** [doublesymmetry/react-native-track-player#2120]
**Describe the Bug**
Updating metadata with duration, using ```updateNowPlayingMetadata```, ```updateMetadataForTrack``` does not update the notification.
title and artist will be updated. Tracks obtained through ```getActiveTrack``` will also have a duration.

**Steps To Reproduce**
Just add a duration to the track and update the metadata.
Notifications will not be updated regardless of whether the track has a duration or not.

**Code To Reproduce**
 ``` 
const Progress = await TrackPlayer.getProgress();
const currentTrack = await TrackPlayer.getActiveTrackIndex();
const track = await TrackPlayer.getTrack(currentTrack);
      track.duration = Progress.duration;
      await TrackPlayer.updateMetadataForTrack(
        currentTrack,
        track
      );
```

**Environment Info:**
Track Player ver - "react-native-track-player": "v4.0.0-rc08"
Device - simulator: android 13, real device : galaxy s21 ultra  android 13
OS - Windows 11

**How I can Help**
It's not a problem that occurs in js.
It appears that the problem is occurring in native. However, I don't know how to use java.
Also, the problem occurs starting from "v4.0.0-rc06".

**Comment by lovegaoshi** [comment]
yep, this bug is introduced in https://github.com/doublesymmetry/KotlinAudio/pull/70
and ultimately now in https://github.com/doublesymmetry/KotlinAudio/blob/main/kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt#L193
where getDuration doesnt check overrideAudioItem?.duration

**Comment by Asoundaccount** [comment]
I don't know how to use Java, but somehow it works. thank you!
```NotificationManager.kt```
```
  private fun getDuration(index: Int? = null): Long {
    val mediaItem = if (index == null) player.currentMediaItem else player.getMediaItemAt(index)
    return overrideAudioItem?.duration
     ?: mediaItem?.getAudioItemHolder()?.audioItem?.duration 
     ?: -1
}
```
